src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -363,5 +363,73 @@ public async Task TestCredentialsCopyInAsyncContext()
                 server.Stop();
             }
         }
+
+
+        [Theory]
+        [InlineData(false, false, false)]
```

Comment:
Are all of these test cases meaningful? It seems like there are only four real test cases here:
1. SevenBit sync
2. SevenBit async
3. UTF-8 sync
4. UTF-8 async